rules/test/content/test_rule/plugin.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Introducing Insights for Red Hat OpenShift Container Platform
+node_id:
```

Comment:
looking at other rules, empty values are probably specified like this https://gitlab.cee.redhat.com/ccx/ccx-rules-ocp/-/blob/master/content/external/rules/prometheus_backed_by_pvc/plugin.yaml 
again.. probably not breaking anything (?)

Reply:
not breaking, but I'll update it JIC (someone one day might look at it and think "this is the way")